test/tests.cpp

```diff
@@ -8,15 +8,19 @@
 
 using namespace ReadSpeed;
 
-void RequireFile(const std::string &fname)
+void RequireFile(const std::string &fname, const std::vector<std::string> &branchNames = {"x"})
 {
    if (gSystem->AccessPathName(fname.c_str()) == false) // then the file already exists: weird return value convention
       return;                                           // nothing to do
 
    TFile f(fname.c_str(), "recreate");
    TTree t("t", "t");
-   int x = 42;
-   t.Branch("x", &x);
+
+   int var = 42;
+   for (const auto b : branchNames) {
```

Comment:
avoid string copies:

```suggestion
   for (const auto &b : branchNames) {
```